loopdb/sqlc/migrations/000015_static_address_withdrawals.up.sql

```diff
@@ -0,0 +1,26 @@
+-- withdrawals stores finalized static address withdrawals.
+CREATE TABLE IF NOT EXISTS withdrawals (
+    -- id is the auto-incrementing primary key for a withdrawal.
+    id INTEGER PRIMARY KEY,
```

Comment:
I think it'd be nice to add an `initiated_at` column to this table, so that we can then filter on that on the cli too.

Reply:
done